share/locale/en.json

```diff
@@ -95,6 +95,10 @@
     "gui_settings_tor_bridges_no_bridges_radio_option": "Don't use bridges",
     "gui_settings_tor_bridges_obfs4_radio_option": "Use built-in obfs4 pluggable transports",
     "gui_settings_tor_bridges_obfs4_radio_option_no_obfs4proxy": "Use built-in obfs4 pluggable transports (requires obfs4proxy)",
+    "gui_settings_tor_bridges_meek_lite_amazon_radio_option": "Use built-in meek_lite (Amazon) pluggable transports",
+    "gui_settings_tor_bridges_meek_lite_amazon_radio_option_no_meek_lite_obfs4proxy": "Use built-in meek_lite (Amazon) pluggable transports (requires obfs4proxy)",
```

Comment:
Sorry @irykoon, one last bug I spotted -  this string name:
```
gui_settings_tor_bridges_meek_lite_amazon_radio_option_no_meek_lite_obfs4proxy
```
should be
```
gui_settings_tor_bridges_meek_lite_amazon_radio_option_no_obfs4proxy
```
as that 's what you call it in the settings_dialog.py

Reply:
Thank you so much for your careful review, @mig5 !

Done: https://github.com/micahflee/onionshare/pull/610/commits/61384e67686e800c317b9c664dc157cacdabbbc7